docs/guide/ecosystem/documentation.md

```diff
@@ -1,91 +1,227 @@
+If you are building a Design System and want to showcase the powers of your components, this would probably be your first choice.
 
-### Vue Styledguidist
-https://github.com/vue-styleguidist/vue-styleguidist
+Example: [vue-styleguide](http://rafaelescala.com/vue-styleguide/) or [vue-design-system](https://vueds.com/example/)
+
+It supports all the following vue features:
+
+- Single file components
+- Class style components
+- Functional component templates
+- Template and script commenting (Events, Methods, Props, Slots)
+- JSX & TSX
+- TypeScript & Flow
+- Vuex & Vue-i18n
+- externalized sources for script and template
+- styled components
+
+#### Documentation style
+
+styleguidist uses JSDoc:
+
+```javascript
+props: {
+    /**
+     * The color for the button example
+     */
+    colorExtends: {
+      type: String,
+      default: '#333'
+    }
+  }
+```
+
+#### Pros
+
+- Displays a live editor to try components
+- Meant for design system creation (powers vue-design-system)
+- A strong React community
+
+#### Cons
+
+- Little configurability on output (HTML only)
+- JSDoc can be verbose
+- Scaffolding built in React
 
 ### Vuese
+
 https://github.com/vuese/vuese
 
-### Vuedoc
-https://gitlab.com/vuedoc/md
+Vuese is a documentation oriented.
+It sacrifices a bit of user experience for a lot of integrability.
+If you have to generate your doc to integrate it, this is probably the tool that you want.
 
-### JSDoc Vuejs
-https://github.com/Kocal/jsdoc-vuejs
+Example: https://buptsteve.github.io/markdown-it-vuese/example/demo.html#props
+
+It supports all the following vue features:
+
+- Single file components
+- Class style components
+- Template and script commenting (Events, Methods, Props, Slots)
+- JSX & TSX
+- TypeScript & Flow
+- Vuex & Vue-i18n
+
+#### Documentation style
+
+vuese uses its own documentaion format more compact than JSDoc
+
+```javascript
+props: {
+  // The name of the form
+  name: {
+    type: String
+  }
+}
+```
+
+#### Pros
+
+- Compact documentation
+- Vuepress default output
+- Configurable docute or markdown outputs
+- Entirely made with VueJs
+
+#### Cons
+
+- Static code samples
 
 ### Vue Design System
+
 https://github.com/viljamis/vue-design-system
 
-## Other useful tools
+Vue designs system is a combination of vue-styleguidist and [salesforce-ux/theo](https://github.com/salesforce-ux/theo) that helps teams build a starter kit for their design system. It is highly opinionated. Choose it if you are starting a new project or if you don't want to think about your own naming.
 
-### Storybook
-[Storybook](https://storybook.js.org/) is a development environment for UI components. It allows developers to setup interactive examples of how a UI component can be used. Simply put, you can setup a live demo of a Vue component by passing different props to it, creating a working sandbox.
+vueds documentation is a great read, even if you are not going to use it.
```

Comment:
Maybe add the acronym above, next to the name in brackets, so ppl know what you are talking about?